internal/scripts/installation/config/install_met_env.generic

```diff
@@ -1,59 +1,65 @@
-#Find the directory this script is called from
+# Find the directory this script is called from
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-#Required
-#Directory that is the root of the compile
-export TEST_BASE=$DIR/../
+# Required
+# Directory that is the root of the compile
+export TEST_BASE=${DIR}
 
-#Required
-#Compiler options = gnu, intel, ics, ips, PrgEnv-intel, or pgi
-#Compiler+version can be used for machines using modules e.g. gnu_6.3.0
-export COMPILER=gnu
+# Required
+# Format is compiler_version (e.g. gnu_8.3.0)
+# Compiler options = gnu, intel, ics, ips, PrgEnv-intel, or pgi
+# Version is used for gnu in compilation of BUFRLIB and HDF5
+export COMPILER=gnu_8.3.0
 
-#Required
-#Root directory for creating/untaring met source code - usually same as TEST_BASE
+# Required
+# Root directory for creating/untaring met source code - usually same as TEST_BASE
 export MET_SUBDIR=${TEST_BASE}
 
-#Required
-#The name of the met tarbal usually downloaded with version from dtcenter.org and includes a version
-# example - met-9.0.1.20200423.tar.gz
-#met.tar.gz is used for compiling from cloned github repo
-export MET_TARBALL=met.tar.gz
+# Required
+# The name of the met tarbal usually downloaded with version from dtcenter.org and includes a version
+#  example - v11.1.0.tar.gz
+export MET_TARBALL=v12.0.0.tar.gz
 
-#Required
-#Specifiy if machine useds modules for loading software
+# Required
+# Specify if machine useds modules for loading software
 export USE_MODULES=FALSE
 
-#Root directory of your python install
-export PYTHON_LOC="$(python3-config --prefix)"
+# Root directory of your python install, containing the bin, include, lib, and share directories
+export MET_PYTHON=`python3-config --prefix`
 
-#Directory of your python executable
-export MET_PYTHON=${PYTHON_LOC}
+# Python ldflags created using python3-config
+export MET_PYTHON_LD=`python3-config --ldflags --embed`
 
-#Python ldflags created using python3-config
-export MET_PYTHON_LD="$(python3-config --ldflags)"
+# Python cflags created using python3-config
+export MET_PYTHON_CC=`python3-config --cflags`
 
-#Python cflags created using python3-config
-export MET_PYTHON_CC="$(python3-config --cflags)"
+# Use MAKE_ARGS to sped up the compilation of the external libaries and/or MET
+# MAKE_ARGS can be set "-j #" where # is replaced with the number of
+# cores to use (as an integer) or to simply "-j" to use all available cores.
+export MAKE_ARGS=-j
 
-#64 bit machine or not
-export SET_D64BIT=FALSE
-
-#General CFLAGS
-export CFLAGS="-Wall -g"
-
-#General CXXLAGS
-export CXXFLAGS="-Wall -g"
-
-#Normally should be omitted or set to TRUE - only used if building from github repo
-export USE_MET_TAR_FILE=FALSE
-
-#If you've already compiled these and don't need to compile them again, set the following
+# If you've already used the script to compile these libaries and don't need to
```

Comment:
> What should be done if the libraries aren't installed? Leave them commented out?

Yes.

> if a user already has the libraries installed, they can leave the statements commented out, ONLY IF they're installed in the {EXTERNAL_LIBS} location.

If the libraries are commented out, the script will always attempt build the libraries in the external_libs directory. If the libraries are installed somewhere other than the external_libs directory and the user would like to use those libraries, they will want to uncomment the line and provide the path.  If the libraries are installed in the external_libs directory and the user would like to use those libraries, they will want to uncomment the line and leave it as-is.  If the libraries are not installed and the user would like the script to install them, they should leave the lines commented out.  

Phew, that's kind of a mouthful, let me try to say it better:

Uncomment the external library variables if the libraries are installed in any location and the user would like to make use of those pre-installed libraries.  If the libraries are in the external_libs dir, uncomment and do nothing.  If the libraries are not in the external_libs dir, uncomment and change the paths to the appropriate location.

Leave the external library variables commented out if the user would like the library/libraries installed in the external_libs subdirectory.

Reply:
That clears it up a bit, thanks. Given that truth, I would suggest a slight change to the wording to reflect closer to what you've just said,

> If users have already installed these libraries and would like to make use of them, uncomment out the export statements. If those pre-existing libraries are in the external_libs directory, no further edits need occur; however, users that have the pre-existing libraries not in the external_libs directory will need to update the paths to the appropriate location.

Does that convey the necessary information well?

Reply:
@j-opatz, yes that conveys the necessary information well. Thank you for simplifying it! I appreciate it!

I made one slight change to that wording the in the update to the documentation.  Instead of "no further edits need occur", I said, "no further edits are needed".  If you prefer the original wording that is fine, and please feel free to change it.